src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -51,11 +51,15 @@ public InputOutputColumnPair(string outputColumnName, string inputColumnName = n
     public static class TransformExtensionsCatalog
     {
         /// <summary>
-        /// Copies the input column to another column named as specified in <paramref name="outputColumnName"/>.
+        /// Creates a <see cref="ColumnCopyingEstimator"/> that copies the data from the column specified in <paramref name="inputColumnName"/>
+        /// to a new column: <paramref name="outputColumnName"/>.
         /// </summary>
+        /// <remarks>For more information on this estimator see <see cref="ColumnCopyingEstimator"/>.</remarks>
```

Comment:
> on this estimator [](start = 41, length = 18)

let's just drop 'on this estimator' b/c this is the extension method. I think 'for more information see ...' is enough